packages/react-composites/src/composites/localization/locales/en-US/strings.json

```
@@ -68,7 +68,10 @@
     "holdScreenLabel": "You're on hold",
     "openDtmfDialpadLabel": "Show dialpad",
     "dtmfDialpadPlaceholderText": "Enter number",
-    "outboundCallingNoticeString": "Calling..."
+    "outboundCallingNoticeString": "Calling...",
+    "participantJoinedNoticeString": "{displayName} joined",
```

this is actually a list of names now?
```suggestion
    "participantJoinedNoticeString": "{displayNames} joined",
```